service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -146,9 +154,43 @@ private boolean registerSchemas() {
         if (!srClient.registerSchema(microservice.getServiceId(), schemaId, content)) {
           return false;
         }
+      } else {
+        if (!checkSchemaSummary(schemaId, content)) {
```

Comment:
all existing schema, must invoke SC two times:
1.check if exists
2.check signature

can we merge it to be a single invoke?

and you already modify name to signature, why name summary here?